### ISSUE TITLE
Add a "problems logging in?" link

### ISSUE THREAD

**Issue by andymckay** [mozilla/addons-server#3653]
We now bounce users over to Firefox Accounts so that when a problem occurs, there is no specific AMO details we can show them. An example is all the people saying "you deleted my account", when we did no such thing, they just have to create a Firefox Account.

Wondering if we could give some choices at the top like:

<img width="676" alt="screenshot 2016-09-30 08 55 44" src="https://cloud.githubusercontent.com/assets/74699/18998008/b6a3d3d6-86eb-11e6-97f5-79525d4ae1aa.png">

And link to SUMO or something. With some details.

**Comment by pwalm** [comment]
Yep, I can build this into AMO desktop screens. Do we need this on Devhub too?

**Comment by eviljeff** [comment]
@pwalm I'm going to keep this issue open for devhub - is there already an issue open for making changes for new frontend we can piggyback this on?
